### ISSUE TITLE
Why FBnet multiply two loss together? Why the alpha purpose?

### ISSUE THREAD

**Issue by andyhahaha** [AnnaAraslanova/FBNet#8]
I can't understand the loss function of FBnet.
There are cross-entropy loss and latency loss. Usually multiple loss weighted sum together become total loss. Why FBnet multiply them together? 
Besides, there is an alpha scale the total loss. Isn't scale the total loss equal to scale the learning rate?  What is its purpose?

**Comment by lxiongh** [comment]
The same question confuses me
